MahApps.Metro/Controls/Dialogs/ProgressDialogController.cs

```diff
@@ -12,16 +12,27 @@ public class ProgressDialogController
         private ProgressDialog WrappedDialog { get; set; }
         private Func<Task> CloseCallback { get; set; }
 
-        /// <summary>
-        /// Gets if the wrapped ProgressDialog is open.
-        /// </summary>
-        [Obsolete("Use the Closed event instead")]
-        public bool IsOpen { get; private set; }
+       
 
         /// <summary>
-        /// This event is raised when the associated <see cref="ProgressDialog"/> was closed.
+        /// This event is raised when the associated <see cref="ProgressDialog"/> was closed programmatically.
         /// </summary>
         public event EventHandler Closed;
+   
+        /// <summary>
+        /// This event is raised when the associated <see cref="ProgressDialog"/> was cancelled by the user.
+        /// </summary>
+        public event EventHandler Cancelled;
```

Comment:
Typical Americanism. Everywhere else in the world it's cancelled (with double l).

http://grammarist.com/spelling/cancel/